### PR TITLE
LPS-146244 Color field predefined value is not recognized on Form Rules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Related Issue: https://issues.liferay.com/browse/LPS-146244

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
LGTM 

/cc @alinedoleron

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/color/ColorDDMFormFieldValueAccessor.java [hunk 1]
```diff
@@ -38,6 +39,11 @@
 public class ColorDDMFormFieldValueAccessor
 	implements DDMFormFieldValueAccessor<String> {
 
+	@Override
+	public IntFunction<String[]> getArrayGeneratorIntFunction() {
+		return String[]::new;
+	}
+
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/color/ColorDDMFormFieldValueAccessor.java:46]: LGTM

/cc @alinedoleron

**Review by carolmariaabb — APPROVED** [review]

**Pull request closed without merging**